docker/Dockerfile.static

```diff
@@ -24,14 +24,15 @@ ENV PYTHONPATH /usr/local/lib/python3.9/site-packages
 
 WORKDIR /usr/src/app
 
-COPY b3lb/requirements.txt \
-     b3lb/manage.py \
+COPY b3lb/requirements.txt .
+RUN pip3 install --no-cache-dir --prefix=/usr/local -r requirements.txt
+
+COPY b3lb/manage.py \
```

Comment:
Please do not add another container image layer if not required.

Reply:
Same as for `Dockerfile.dev`